src/libraries/System.Runtime.WindowsRuntime/System.Runtime.WindowsRuntime.sln

```diff
@@ -26,6 +26,10 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{C4854B44-ABFE-4BB5-8F89-F35FE6201338}.Release|Any CPU.Build.0 = Release|Any CPU
```

Comment:
Do we know why VS hangs without this?  Are the right folks aware of that bug?

Reply:
Yes there is a similar issue opened in the VS https://github.com/dotnet/project-system/issues/5878

The fix for vs project was same here https://github.com/dotnet/runtime/pull/31950